### PR TITLE
Trim the string, not the promise

### PR DESCRIPTION
**Pull request by imurchie**
Fixes https://github.com/appium/appium/issues/6978

### CODE REVIEW

File: lib/tools/adb-commands.js [hunk 1]
```diff
@@ -424,7 +424,7 @@ methods.getPIDsByName = async function (name) {
     if (name.length > 15) {
       name = name.substr(name.length - 15);
     }
-    let stdout = await this.shell(["ps"]).trim();
+    let stdout = (await this.shell(["ps"])).trim();
```
Comment (jlipps) [inline lib/tools/adb-commands.js]: 🤕

**Comment by lelandrichardson** [comment]
When is this planned to get released?  Thanks!

**Comment by imurchie** [comment]
Published in `appium-adb@2.6.18`.

**Comment by lelandrichardson** [comment]
Thanks!

**Comment by Vishnu1067** [comment]
I still have a problem

Appium - v1.6.0
adb - 2.6.18
node - v6.8.1

When i install with npm install -g appium --no-shrinkwrap, there is an error like  

`ERROR in ./Scripts/Performance.ts
Module build failed: TypeError: Path must be a string. Received undefined
    at assertPath (path.js:7:11)
    at Object.dirname (path.js:697:5)`

[https://github.com/webpack/webpack/issues/2463](url)

Reference: https://github.com/webpack/webpack/issues/2463

Is there new release happening for adb again? Please help me to resolve this problem.

**Comment by imurchie** [comment]
@Vishnu1067 When is this error happening? What is the context?

**Comment by Vishnu1067** [comment]
ADB error trim is not a function as mentioned in the issue.

https://github.com/appium/appium/issues/6978.

I use node 6.8.1 and also installed latest adb version. I don't want to degrade node to 5xxx... please instruct me how to resolve the problem.

**Comment by imurchie** [comment]
I mean the error you mention here: https://github.com/appium/appium-adb/pull/180#issuecomment-255006550

**Comment by Vishnu1067** [comment]
@imurchie When I install npm install -g --no-shrinkwrap

I got an error while executing test as mentioned in below id,
https://github.com/webpack/webpack/issues/2463

**Comment by imurchie** [comment]
Can you provide the output? The full logs from running the npm install?

**Pull request merged**